### PR TITLE
Added signature text to address issue 42

### PR DESCRIPTION
**Pull request by mikelodder7**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/did-spec/pull/91.html" title="Last updated on Jun 25, 2018, 1:20 PM GMT (1f49fee)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/did-spec/91/f6ebf04...1f49fee.html" title="Last updated on Jun 25, 2018, 1:20 PM GMT (1f49fee)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1933,6 +1934,7 @@ <h3>Proving Ownership of a DID and DID Document</h3>
   <li>
 The owner of the DID Document controlled the private key used for the
 signature at the time the signature was generated.
+explain
```
Comment (msporny) [inline index.html]: Is this PR a WIP? Mostly wondering if the "explain" text is an errant copy/paste or placeholder?

**Comment by mikelodder7** [comment]
It looks like a copy/paste error I’ll fix it

File: index.html [hunk 2]
```diff
@@ -1922,6 +1922,7 @@ <h2>Binding of Identity</h2>
 <h3>Proving Ownership of a DID and DID Document</h3>
 
 <p>
+Signatures are one method to allow DID Documents to be cryptographically verifiable.<br>
```
Comment (msporny) [inline index.html]: Don't use &lt;br>s to do layout... either use a new &lt;p> tag or do things in a list via &lt;ul> or &lt;ol>s.

File: index.html [hunk 3]
```diff
@@ -1953,7 +1954,9 @@ <h3>Proving Ownership of a DID and DID Document</h3>
 </ol>
 
 <p>
-It should be noted that this process proves ownership of a DID and DID Document regardless of whether the DID Document is signed.
+It should be noted that this process proves ownership of a DID and DID Document regardless of whether the DID Document is signed.<br>
+Signatures on DID Documents are optional. DID Method Specs should explain and specify their implementation if applicable.<br>
+It is recommended to combine timestamps with signatures.
```
Comment (msporny) [inline index.html]: Use all caps for RECOMMENDED to trigger the auto-markup feature for ReSpec. All RFC keywords should be in all caps.... MUST, SHOULD, RECOMMENDED, etc.

**Comment by msporny** [comment]
Rebased to fix merge conflicts, merging. Thanks @mikelodder7!

**Pull request merged**